workspace/TS100/configuration.h

```diff
@@ -99,10 +96,9 @@
   #define MAX_POWER_LIMIT         30        //
   #define POWER_LIMIT_STEPS       2
 
-  /**
-   *  TIP_GAIN =  TIP_GAIN/1000 == uV per deg C constant of the tip
-   */
   #define OP_AMP_Rf               180*1000  //  180  Kilo-ohms -> From schematic, R6
   #define OP_AMP_Rin              2000      //  2.0  Kilo-ohms -> From schematic, R3
-  #define TIP_GAIN                115
+  #define TIP_GAIN_ADJUSTMENT     100 / 349 //  due to different OP_AMP_R f/in with TS100 we have to use this coeff while converting uV to temp
```

Comment:
Dont use `TIP_GAIN_ADJUSTMENT` If you can. It's a magic number that can be derived in the source code and compiler will optimise it (often somewhat better)

 It would be better to in the function derive these values by having the #if / #else / #endif block include code in both sides that directly uses OP_AMP_* #defines to compute the gain in the correct location.

Sorry to pick on you about this, i shouldn't have left the previous code merge that got rid of the use of these #defines.